extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -347,10 +350,23 @@ export class AmpLightboxGallery extends AMP.BaseElement {
       this.buildCarouselSlides_(list);
       return this.mutateElement(() => {
         this.carouselContainer_.appendChild(this.carousel_);
+        this.toggleNavControls_(list.length);
       });
     });
   }
 
+  /**
+   * @param {number} noOfChildren
+   * @private
+   */
+  toggleNavControls_(noOfChildren) {
+    if (noOfChildren > 1) {
+      this.controlsContainer_.classList.remove('i-amphtml-lbg-single');
```

Comment:
nit: You can use `classList.toggle` e.g.

```js
this.controlsContainer_.classList.toggle('i-amphtml-lbg-single', noOfChildren <= 1);
```

Reply:
super optional nit: `childCount` is easier to read since it doesn't use an abbreviation

Reply:
Do you plan to use the CSS class `'i-amphtml-lbg-single` for other purposes? If not, you can toggle the internal class `i-amphtml-hidden` that AMP already uses to hide elements.

Reply:
Oh... hmm. I'll need to toggle it for three elements I guess.

Reply:
That's true, that could introduce complexity. I'm fine with the CSS how it is.

Reply:
FYI I just checked though. We don't actually have an `i-amphtml-hidden` css class. We have an `amp-hidden` class. It doesn't have default behavior, but each component defines behavior for it separately...

Reply:
Ah I grepped too quickly!